srcpkgs/python3-evdev/template

```diff
@@ -1,17 +1,24 @@
 # Template file for 'python3-evdev'
 pkgname=python3-evdev
 version=1.6.0
-revision=2
+revision=3
 build_style=python3-module
 hostmakedepends="python3-setuptools"
 makedepends="python3-devel"
+depends="python3"
+checkdepends="python3-pytest"
 short_desc="Python3 bindings for the Linux input subsystem"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="BSD-3-Clause"
 homepage="https://github.com/gvalkov/python-evdev"
 distfiles="https://github.com/gvalkov/python-evdev/archive/v${version}.tar.gz"
 checksum=85877121b46e0eacf390c1ee4a07e6f5fcb3c603c1649efcccdd554f2fe0c3ab
 
+pre_check() {
+	python3 setup.py build_ext --inplace
+	rm tests/test_uinput.py # permission issue with /dev/uinput
```

Comment:
Check can run multiple times, but rm will only succeed the first time because the file will be deleted then. Use -f or maybe better, add --deselect to pytest args instead.
```suggestion
	PYTHONPATH="$(cd build/lib* && pwd)" pytest
	rm -f tests/test_uinput.py # permission issue with /dev/uinput
```

Reply:
Huh! I actually tried that before submitting but with `python3 -m pytest`, as seen in the python3-module build_style, which didn't work. So, under `do_check()`:

`PYTHONPATH="$(cd build/lib* && pwd)" pytest --ignore tests/test_uinput.py` -> works

`PYTHONPATH="$(cd build/lib* && pwd)" python3 -m pytest --ignore tests/test_uinput.py` -> fails with import issues.

Reply:
Yeah, this is a bit weird. Just yesterday this came up in #xbps on IRC. `python3 -m pytest` and `pytest` are different in how they set PYTHONPATH.

Reply:
I see, thanks for the info.